e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide to Running Tests
```

Comment:
These instructions are Snapp specific, maybe incuded a sub-section so that is clear. We can add the StableX specific steps in another PR I think.